docs/marks/link.md

```diff
@@ -0,0 +1,179 @@
+<script setup>
+
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+import * as topojson from "topojson-client";
+import {computed, shallowRef, onMounted} from "vue";
+import income from "../data/income-gender.ts";
+import metros from "../data/metros.ts";
+
+const xy = {x1: -122.4194, y1: 37.7749, x2: 2.3522, y2: 48.8566};
+const gods = ["Chaos/Gaia/Mountains", "Chaos/Gaia/Pontus", "Chaos/Gaia/Uranus", "Chaos/Eros", "Chaos/Erebus", "Chaos/Tartarus"];
+const world = shallowRef(null);
+const land = computed(() => world.value ? topojson.feature(world.value, world.value.objects.land) : null);
+
+onMounted(() => {
+  d3.json("../data/countries-110m.json").then((data) => (world.value = data));
+});
+
+</script>
+
+# Link mark
+
+The **link mark** draws straight lines between two points [**x1**, **y1**] and [**x2**, **y2**] in quantitative dimensions. It is similar to the [arrow mark](./arrow.md), except it draws a straight line—or geodesic when used with a [spherical projection](../features/projections.md).
+
+For example, the chart below shows the rising inequality (and population) in various U.S. cities from 1980 to 2015. Each link represents two observations of a city: the city’s population (**x**) and inequality (**y**) in 1980, and the same in 2015. The link’s **stroke** redundantly encodes the change in inequality: red indicates rising inequality, while blue (there are only four) indicates declining inequality.
+
+:::plot defer
+```js
+Plot.plot({
+  grid: true,
+  inset: 10,
+  x: {
+    type: "log",
+    label: "Population →"
+  },
+  y: {
+    label: "↑ Inequality",
+    ticks: 4
+  },
+  color: {
+    scheme: "BuRd",
```

Comment:
```suggestion
    scheme: "BuRd",
    symmetric: false,
```

without the symmetric option, we don't see any blue link.

Reply:
I tried that, but I don’t like the way the legend looks. I think they are still technically “blue” even though they are a very very pale blue that is almost indistinguishable from white.

<img width="662" alt="Screenshot 2023-04-17 at 3 43 23 PM" src="https://user-images.githubusercontent.com/230541/232626066-f4542288-85e6-439a-bac5-9982b649c0d2.png">